src/app/zap-templates/zcl/zcl.json

```diff
@@ -744,7 +744,6 @@
             "AllocatedVideoStreams",
             "AllocatedAudioStreams",
             "AllocatedSnapshotStreams",
-            "RankedVideoStreamPrioritiesList",
```

Comment:
StreamUsagePriorities needs to be added back.

Reply:
As per comment for zcl-with-test-extensions, removed everything that isn't a Struct